src/components/TodoListItem.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
```

Comment:
Great job implementing the delete and add functionality. To increase your UX exponentially I suggest touching up your UI. There is a lot of dead space between a list item and the delete button, which makes it hard to align one to another.  A visual connection would be fantastic.